### PR TITLE
WFLY-4439 NPE when deployment contains a *-ds.xml in domain

### PR DESCRIPTION
**Pull request by maeste**

### CODE REVIEW

**Comment by jamezp** [comment]
This is the wrong JIRA id too. I think it should be https://issues.jboss.org/browse/WFLY-4339

**Comment by maeste** [comment]
rebased and fixed JIRA id

File: connector/src/main/java/org/jboss/as/connector/services/driver/registry/DriverRegistryImpl.java [hunk 1]
```diff
@@ -81,15 +82,17 @@ public void unregisterInstalledDriver(InstalledDriver driver) {
 
     @Override
     public Set<InstalledDriver> getInstalledDrivers(String profileName) {
-        String profile = profileName != null ? profileName : "nullProfile";
+        String profile = profileName != null ? profileName : "default";
 
         return Collections.unmodifiableSet(Collections.synchronizedSet(new HashSet<InstalledDriver>(drivers.get(profile).values())));
```
Comment (bstansberry) [inline connector/src/main/java/org/jboss/as/connector/services/driver/registry/DriverRegistryImpl.java]: This looks like it would NPE as well.

**Comment by maeste** [comment]
@bstansberry fixed and squashed

**Pull request merged**